app/homepage/index.html

```diff
@@ -7,6 +7,11 @@
         <link rel="stylesheet" href="./styles.css">
     </head>
     <body>
+        <div id="navbar">
+            <a class="active" href="/">Home</a>
+            <a id="login-link" href="/login">Login</a>
+            <a id="register-link" href="/register">Register</a>
+        </div>
```

Comment:
You've already said to me about the `<nav>` element :)

Reply:
Done :)